lib/printfrr.h

```diff
@@ -275,6 +275,17 @@ struct va_format {
 	va_list *va;
 };
 
+/* len of "xx.xxxx.xxxx.xxxx.xxxx.xxxx.xxxx.xxxx.xxxx.xxxx.xx" + '\0' */
+#define ISO_NET_SIZE 51
+/* %pIN extension need to define an ISO Network address structure */
+struct iso_net {
```

Comment:
identical to this in isis_common.h 

Do not duplicate the structure

```
/*
 * Area Address
 */
struct area_addr {
	uint8_t addr_len;
	uint8_t area_addr[20];
};
```

Reply:
Yes. I know. My concern is that we need to define a structure to print the ISO Nework Address which should be common to all daemons in one hand, and in an other hand, I would not impact too much the ISISd code.

isis_common.h is included in many isis_xxx files, while the printfrr.h is a few of them. To share a common structure, I need to find a good place to define this structure.